### PR TITLE
Chels quick order

### PR DESCRIPTION
**Pull request by chelseawinfree**

### CODE REVIEW

File: symantecssl/models.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            xml_node.find('.//m:Email', NS)
+        )
+        contact.title = utils.get_element_text(
+            xml_node.find('.//m:Title', NS)
+        )
+
+        return contact
+
+    def serialize(self, element_name):
+        """
```
Comment (jmvrbanac) [inline symantecssl/models.py]: Summary line? If you don't have one, I would remove the extra blank line.

File: symantecssl/models.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        fax = etree.SubElement(ele, 'Fax')
+        fax.text = self.fax
+
+        return ele
+
+    def set_contact_info(
+            self, first_name, last_name, phone, email, title,
+            org_name=None, address_one=None, address_two=None, city=None,
+            region=None, postal_code=None, country=None, fax=None
+    ):
```
Comment (jmvrbanac) [inline symantecssl/models.py]: Move this back up to line 139

File: symantecssl/models.py [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        """
+
+        :param element_name: contact element type. Limited to Admin, Tech, and
+         Billing.
+        :return: the contact element that is to be used for request.
+        """
+
+        ele = etree.Element(element_name)
+        first_name = etree.SubElement(ele, 'FirstName')
+        first_name.text = self.first_name
```
Comment (jmvrbanac) [inline symantecssl/models.py]: Something you might consider is creating a function in utils to create a SubElement and add the text. That should remove some duplication.

**Comment by jmvrbanac** [comment]
:+1:

**Comment by sheyman** [comment]
Love it!

**Pull request merged**